instrumentation/opentelemetry-instrumentation-api/javaagent/build.gradle.kts

```diff
@@ -2,10 +2,28 @@ plugins {
   id("otel.javaagent-instrumentation")
 }
 
+// TODO (trask) muzzle version check doesn't work for two reasons
+//  * all of the 1.0.0+ opentelemetry-instrumentation-api releases are alpha so they are skipped
+//  * I think dependency substitution is applied to the dependency in the muzzle directive,
+//    causing this error:
```

Comment:
Yes it is: I encountered the same thing when I added the `testOldServerSpan` test suite, I think you can probably fix the muzzle problem in a similar way to that.